src/Ml.Cli.WebApp/ClientApp/src/Server/Project/Project.service.js

```diff
@@ -32,4 +32,9 @@ export const fetchCreateProject = fetch => async newProject =>
         body: JSON.stringify(newProject),
     });
 
+export const fetchDeleteProject = fetch => async (id) =>
+    fetch(`projects/delete/${id}`, {
```

Comment:
**suggestion**: Plug OAUTH / OPENID CONNECT + CORS HEADER

Reply:
It's already done :)